web/src/pages/Cases/CaseDetails/Appeal/Classic/Fund.tsx

```diff
@@ -53,21 +74,25 @@ const Fund: React.FC = () => {
           }}
           placeholder="Amount to fund"
         />
-        <StyledButton
-          disabled={isDisconnected || isSending || !balance || parsedAmount > balance.value}
-          text={isDisconnected ? "Connect to Fund" : "Fund"}
-          onClick={() => {
-            if (fundAppeal) {
-              setIsSending(true);
-              wrapWithToast(fundAppeal!())
-                .then(() => {
-                  setAmount("");
-                  close();
-                })
-                .finally(() => setIsSending(false));
-            }
-          }}
-        />
+        {chain && chain.id === DEFAULT_CHAIN ? (
```

Comment:
abstract this into `ensureChain` function, that receives a ReactNode and wraps it with the check